src/libraries/System.Data.Common/src/System/Xml/DataSetMappper.cs

```diff
@@ -204,18 +202,18 @@ internal DataTable GetTableSchemaForElement(XmlElement elem)
         //     XmlElement e = be;
         //     GetRowFromElement( be ); // Calls GetRowFromElement( XmlBoundElement )
         //     GetRowFromElement( e );  // Calls GetRowFromElement( XmlElement ), in spite of e beeing an instance of XmlBoundElement
-        internal DataRow GetRowFromElement(XmlElement e) => (e as XmlBoundElement)?.Row;
+        internal DataRow? GetRowFromElement(XmlElement? e) => (e as XmlBoundElement)?.Row;
```

Comment:
Interesting. This line was already using nullability even though it was disabled for this file.

Reply:
Which lines are you referring to?

Reply:
Lines 195 and 205.

Reply:
`be.Row?.Table;`

`(e as XmlBoundElement)?.Row;`

Reply:
But those aren't null annotations; they are just using the [null conditional operator](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/member-access-operators#null-conditional-operators--and-), which exists since C# 6.

Reply:
That's interesting. Thanks for clarifying.